buku

```diff
@@ -5835,6 +5838,8 @@ def main(argv=sys.argv[1:], *, program_name=os.path.basename(sys.argv[0])):
     desc_in = None
     pipeargs = []
     colorstr_env = os.getenv('BUKU_COLORS')
+    allowed_default_schemes = (SCHEME_HTTP, SCHEME_HTTPS)
```

Comment:
…I don't believe this is necessary at all; if a user wants to specify `ftp` here or whatnot, there's zero reason to stop him :sweat_smile:

Also, why are you declaring a constant value locally within a function?